src/NServiceBus.TransactionalSession.AcceptanceTests/When_using_outbox.cs

```diff
@@ -74,10 +75,39 @@ public async Task Should_send_immediate_dispatch_messages_even_if_session_is_not
             Assert.True(result.MessageReceived);
         }
 
+        [Test]
+        public async Task Should_make_it_possible_float_ambient_transactions()
+        {
+            var result = await Scenario.Define<Context>()
+                    .WithEndpoint<AnEndpoint>(s => s.When(async (_, ctx) =>
+                    {
+                        using var scope = ctx.ServiceProvider.CreateScope();
+                        using var transactionalSession = scope.ServiceProvider.GetRequiredService<ITransactionalSession>();
+
+                        await transactionalSession.Open(new CustomTestingPersistenceOpenSessionOptions
+                        {
+                            UseTransactionScope = true
```

Comment:
Should we provide `EnlistInExistingTransactionScope` as well?

Reply:
Discussed it with @tmasternak and we have concluded we currently don't need to support this additional feature.